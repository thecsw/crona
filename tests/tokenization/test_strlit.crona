"this is a good string"
"unterminated string
"unterminated string\"
"bad escape char \u"
"bad escape \u and unterminated
"unterminated string with bad escape \e \"
"backslash followed by space: \ is not allowed"