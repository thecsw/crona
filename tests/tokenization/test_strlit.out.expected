STRINGLIT:"this is a good string" [1,1]
EOF [7,48]